player/tests/data/pipeline-statistics-query.ron

```diff
@@ -1,5 +1,5 @@
 (
-    features: 0x0000_0000_0000_0100, // PIPELINE_STATISTICS_QUERY
+    features: 0x0000_0001_0000_0000, // PIPELINE_STATISTICS_QUERY
```

Comment:
Maybe we can use the new `Serialize`/`Deserialize` derives that use a textual representation (in a separate PR).